Components/badges.css

```diff
@@ -37,8 +37,23 @@
 .badge-success{
     background: var(--success);
 }
+.badge-avatar-online{
+    width: 10px;
```

Comment:
I think rems can be used instead of px

Reply:
Only used here as width of this particular unit is required for this badge irrespective of browser/screen orientation